src/main/default/classes/TriggerContext.cls

```diff
@@ -0,0 +1,41 @@
+// https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_triggers_context_variables.htm
+public class TriggerContext {
```

Comment:
i wonder what would happen if we shadowed the system variable

in theory devs could work with `Trigger.*` like they normally do, and we'd just need to update this class to use `System.Trigger.*` to pull the correct namespace ...

Reply:
well nm, if we namespace this you'd have to do `ccc.Trigger.*`

Reply:
it won't let you shadow it.  `Trigger` is reserved (I had the same thought)